install/martypc.toml

```diff
@@ -152,14 +152,14 @@ video = "CGA"
 # "None"  - No hard disk controller will be present
 # "Xebec" - Emulates the IBM/Xebec 20MB Fixed Disk Controller
 
-hdc = "None"
-#hdc = "Xebec"
+#hdc = "None"
+hdc = "Xebec"
```

Comment:
I didn't make this the default option because it requires a ROM I do not provide in the distribution